dist/aw-parser.amd.js

```diff
@@ -182,233 +328,117 @@ parser.parse = function(original_script, cfg) {
         script = script.replace(/ {0,1}\[\[/g, ' /*').replace(/\]\] {0,1}/g, '*/');
     }
 
-    var lines = script.split(/\r\n|\r|\n/);
-
-    var create_token = function(text, cursor, line) {
-        return h.create_token({
-            text: text.trim(),
-            start: cursor,
-            end: cursor + text.length - 1 + new_line_length,
-            line: line
-        });
-    };
-
-    var lines_length = lines.length,
+    var lines = script.split(/\r\n|\r|\n/),
+        lines_length = lines.length,
         current = 0,
-        scene_number = 1,
-        match, text, last_title_page_token,
+        text,
         token, last_was_separator = false,
-        token_category = 'none',
-        last_character_index,
-        dual_right,
-        state = 'normal',
-        cache_state_for_comment,
-        nested_comments = 0,
-        title_page_started = false;
-
-
-    var reduce_comment = function(prev, current) {
-        if (current === '/*') {
-            nested_comments++;
-        } else if (current === '*/') {
-            nested_comments--;
-        } else if (!nested_comments) {
-            prev = prev + current;
-        }
-        return prev;
-    };
-
-    var if_not_empty = function(a) {
-        return a;
-    };
+        cache_state_for_comment;
 
     for (var i = 0; i < lines_length; i++) {
         text = lines[i];
 
         // replace inline comments
-        text = text.split(/(\/\*){1}|(\*\/){1}|([^\/\*]+)/g).filter(if_not_empty).reduce(reduce_comment, '');
+        text = text.split(/(\/\*){1}|(\*\/){1}|([^\/\*]+)/g).filter(if_not_empty).reduce(this._reduce_comment, '');
```

Comment:
We need to bind `_reduce_comment` to make it work (otherwise text between /* .. */ will stay in the script): 
`this._reduce_comment.bind(this)`